src/GitHub.InlineReviews/Services/PullRequestSessionService.cs

```diff
@@ -353,6 +357,42 @@ public virtual async Task<PullRequestDetailModel> ReadPullRequestDetail(HostAddr
             return result;
         }
 
+        private async Task<LastCommitAdapter> GetPullRequestLastCommitAdapter(HostAddress address, string owner, string name, int number)
```

Comment:
Nit: `private` not needed. Also private methods should go after `public`/`protected`.